models/models.js

```diff
@@ -1,13 +1,20 @@
 const db = require ("../db/connection")
+const fs = require("fs/promises");
 
 const selectTopics = () => {
     return db.query(`SELECT * FROM topics;`).then((result) => {
         return result.rows
     });
-}
+};
 
+const selectEndpoints = async () => {
+const endpoints = `./endpoints.json`;
+const endpointsString = await fs.readFile(endpoints, "utf-8");
+return JSON.parse(endpointsString)
```

Comment:
This technically isn't a model as it is interacting with our local files rather than making a request to a db. We could handle this in the controller rather than in a model